components/token-operation/HotWalletBroadcastTokenOperation.tsx

```diff
@@ -105,9 +108,19 @@ export default function HotWalletBroadcastTokenOperation({
         scriptPublicKey: JSON.parse(P2SHEntry.scriptPublicKey.toString()),
         blockDaaScore: P2SHEntry.blockDaaScore.toString(),
         outpoint: JSON.parse(P2SHEntry.outpoint.toString()),
-      };
-
-      const revealTxId = await account.signAndBroadcastTx([], {
+      } satisfies Entry;
+
+      const paymentOutputs: PaymentOutput[] =
+        networkId === NetworkType.Mainnet
+          ? [
+              {
+                address: FORBOLE_PAYOUT_ADDRESS,
+                amount: FORBOLE_FEES[opData.op as OpForboleFeesKey].toString(),
```

Comment:
If amount is 0, then the transaction will fail because of KIP009 requiring minimum amount of utxo to be about 0.02KAS.

Reply:
589b76a64bf98bbf31e82e2bc4747c4bba959667